src/layer.rs

```diff
@@ -41,7 +41,11 @@ impl LayerSet {
     ///
     /// The `glyph_names` argument allows norad to reuse glyph name strings,
     /// reducing memory use.
-    pub fn load(base_dir: &Path, glyph_names: &NameList) -> Result<LayerSet, Error> {
+    pub fn load(
+        base_dir: &Path,
+        glyph_names: &NameList,
+        request: &crate::DataRequest,
```

Comment:
I would not include `DataRequest` as part of the base API here, especially if it is only being used to thread through whether or not we load the glyph libs?